src/main/scala/com/fortysevendeg/translatebubble/ui/components/BubbleView.scala

```diff
@@ -67,67 +62,60 @@ class BubbleView(context: Context, attrs: AttributeSet, defStyleAttr: Int)(impli
     anim
   }
 
-  addView(bubble)
-  addView(loading)
+  runUi(this <~ vgAddViews(Seq(bubble, loading)))
 
-  def init(paramsBubble: WindowManager.LayoutParams) {
+  def init(paramsBubble: WindowManager.LayoutParams): Unit = {
```

Comment:
I'm almost sure that the good one is the previous one (no return declaration and no equal sign) but it's good for me